piksi_tools/console/sbp_relay_view.py

```diff
@@ -312,12 +316,13 @@ class SbpRelayView(HasTraits):
                       show_label=False, resizable=True, padding=15),
                  spring,
                ),
+             visible_when='show_networking',
              ),
              spring
            )
   )
 
-  def __init__(self, link, device_uid=None, base=DEFAULT_BASE, 
+  def __init__(self, link, show_networking=False, device_uid=None, url='', 
```

Comment:
@denniszollo just double check that the caller of this class has updated its arguments, particularly if it depends on the order of the arguments defined here.